src/pages/ProjectDetails/Rewards/RewardForm/WorkItemSidePanel/OtherWorkForm/RepoSelect.tsx

```diff
@@ -16,7 +16,7 @@ export default function RepoSelect({ repos, repo, setRepo }: Props) {
   const { T } = useIntl();
 
   return (
-    <div className="relative flex w-48 flex-col gap-2">
+    <div className="relative flex max-w-sm flex-col gap-2">
```

Comment:
```suggestion
    <div className="relative flex max-w-sm flex-col gap-2 flex-1">
```

This wouldn't work ?